README.md

```diff
@@ -239,6 +240,94 @@ This already goes with how CSP is currently enforcing its rules canonically in t
 
 `X-frames` and `frame-src` allow controlling what is allowed to be loaded into an `iframe`. Regardless of their use to deny loading content in iframes, ways to crete new same origin realms remain available, such as creating an `about:blank` iframe, opening a new tab using `open` API and more.
 
+## [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose, and for what purposes?
+
+This feature allows a website to register JavaScript code to be executed within new realms that fall under its jurisdiction (same origin) before any other JavaScript code gets to run within them.
+Thus, the information it naturally exposes is regarding when a new same origin realm is introudced under the execution environment of the top realm of the website.
+
+> 02.  Do features in your specification expose the minimum amount of information necessary to implement the intended functionality?
+
+Yes, I believe so - no further information is being provided to the website other than what's described under Q#1
+
+> 03.  Do the features in your specification expose personal information, personally-identifiable information (PII), or information derived from either?
+
+No
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+They have nothing to do with such information
+
+> 05.  Does data exposed by your specification carry related but distinct information that may not be obvious to users?
+
+The only distinct information that's being provided to the user that was not possible before is when same origin realms are introduced into the website's execution environment
+
+> 06.  Do the features in your specification introduce state that persists across browsing sessions?
+
+No
+
+> 07.  Do the features in your specification expose information about the underlying platform to origins?
+
+No
+
+> 08.  Does this specification allow an origin to send data to the underlying platform?
+
+The answer is basically no aside for the value of the configured new CSP directive which should be parsed into a URL of a remote resource
```

Comment:
I think this can just be "no"

Reply:
b05732f2dc2275112035657c77bf83ad86217969